Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# 1.4.0 Adaptation

## Changelog:
- Adapted the mod to 1.4.0.
- Unlocked Monarch's Journey hairs and beards for our ethnicities.
  - Wizard's Beard is used by dwarves and vrykul.
  - Miller (female hair) is used by vrykul, dwarves, goblins, gnomes, and trolls.
  - Jeanne d'Arc (female hair) is used by goblins and gnomes.
  - Mullet (male hair) is used by dwarves and vrykul.

# Notes:
- Looks like you can set the exact icon to an activity.
- The buildings got `max_garrison` and `garrison_reinforcement_factor` instead of `garrison`.
- `scheme_progress`... looks like the max progress set to 10 instead of 100.
- Viking raiders spawn with inheritable MaAs

# Tasks:
- [x] Static models not working
- [x] Unlock new hairstyles to new cultures
- [x] Check `add_achievement_flag` doesn't exist

# How to test:
@Warcraft-GoA-Development-Team/testers run the mod on 1.4.0 for some time, maybe even try [A Few Paradox Changes](https://forum.paradoxplaza.com/forum/developer-diary/ck3-dev-diary-63-1-4-0-azure-patch-notes.1478364/).

@Warcraft-GoA-Development-Team/coders open **_Files changed_**, check I haven't left bugs there.
